onchain/rollups/.changeset/chilly-cherries-join.md

```diff
@@ -0,0 +1,7 @@
+---
+"@cartesi/rollups": major
+---
+
+Modified the ERC-20 deposit input.
```

Comment:
This is minor, but should this sentence end with `:`?

Reply:
Yes, thanks. :-)